plugins/woocommerce/patterns/page-coming-soon-minimal-left-image.php

```diff
@@ -0,0 +1,75 @@
+				<div class="wp-block-columns alignfull" style="margin-top:0px;margin-bottom:0px;padding-top:0;padding-right:0;padding-bottom:0;padding-left:0"><!-- wp:column {"verticalAlignment":"top","width":"481px","className":"woocommerce-coming-soon-minimal-left-image__content-image","layout":{"type":"default"}} -->
+					<div class="wp-block-column is-vertically-aligned-top woocommerce-coming-soon-minimal-left-image__content-image" style="flex-basis:481px"><!-- wp:image {"width":"481px","style":{"border":{"radius":"16px"}}} -->
+						<figure class="wp-block-image is-resized has-custom-border"><img src="<?php echo esc_url( $default_image ); ?>" alt="Decorative Image" style="border-radius:16px;width:481px"/></figure>
+						<!-- /wp:image --></div>
+					<!-- /wp:column -->
+
+					<!-- wp:column {"verticalAlignment":"stretch","width":"453px","className":"woocommerce-coming-soon-minimal-left-image__content-text","style":{"spacing":{"blockGap":"0","padding":{"right":"0","left":"0","bottom":"0","top":"53px"}}},"layout":{"type":"default"}} -->
+					<div class="wp-block-column is-vertically-aligned-stretch woocommerce-coming-soon-minimal-left-image__content-text" style="padding-top:53px;padding-right:0;padding-bottom:0;padding-left:0;flex-basis:453px"><!-- wp:group {"style":{"dimensions":{"minHeight":"100%"},"spacing":{"blockGap":"0"}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch","flexWrap":"nowrap","verticalAlignment":"space-between"}} -->
+						<div class="wp-block-group" style="min-height:100%"><!-- wp:heading {"className":"is-style-default","style":{"elements":{"link":{"color":{"text":"#000"}}},"color":{"text":"#000"},"typography":{"fontSize":"38px","lineHeight":"1.19"},"spacing":{"margin":{"bottom":"var:preset|spacing|30"}}},"fontFamily":"heading"} -->
+							<h2 class="wp-block-heading is-style-default has-text-color has-link-color has-heading-font-family" style="color:#000;margin-bottom:var(--wp--preset--spacing--30);font-size:38px;line-height:1.19"><?php echo esc_html__( 'Something big is brewing! Our store is in the works – Launching shortly!', 'woocommerce' ); ?></h2>
+							<!-- /wp:heading -->
+
+<!-- wp:group {"layout":{"type":"constrained","justifyContent":"left","contentSize":"338px"}} -->
+							<div class="wp-block-group"><!-- wp:paragraph {"style":{"color":{"text":"#000"},"elements":{"link":{"color":{"text":"#000"}}},"typography":{"lineHeight":"1.6","letterSpacing":"0px"}}} -->
+								<p class="has-text-color has-link-color" style="color:#000;letter-spacing:0px;line-height:1.6"><?php echo esc_html__( 'Marianne Renoir transforms your home with our curated collection of home decor, bringing inspiration and style to every corner.', 'woocommerce' ); ?></p>
```

Comment:
Could the site name be used here instead of "Marianne Renoir"?

Also, what do you think about using the site tagline (from general settings), if it is not empty? If it is empty, then using this copy.

Reply:
Sure. I'll update the site name and see if it's possible to use the site tagline.

Reply:
I've updated to use the site name and the site tagline (if it's not empty). 352c499edabb46e31776291fbe34e63d68f071cf